onnx/defs/shape_inference.h

```diff
@@ -42,6 +42,7 @@ struct InferenceContext {
   virtual const AttributeProto* getAttribute(const std::string& name) const = 0;
   virtual size_t getNumInputs() const = 0;
   virtual const TypeProto* getInputType(size_t index) const = 0;
+  virtual const TensorProto* getInputInitializer(size_t index) const = 0;
```

Comment:
Thanks for adding this! 

Suggestion: name it as getInput, since it does not have to be an initializer. What we want here is to access input tensor data.

Reply:
Good point - making it getInputData